### DISCUSSION TITLE
Multiple Database Single Code

### DISCUSSION THREAD

**Discussion by senthilsoftware** [prisma/prisma#11701] [Q&A]
I developed API one code for multiple databases with the same schema. while requesting every API it will dynamically change database name and initialize new client every function it worked fine. but nodejs server most of the time facing memory leak error, max connection error, and the server was crashed. when I restart it will work then after one or two days automatically it will down. so every time we face this issue. Kindly give correct suggestions.

**Comment by pantharshit00** [comment]
Hello @senthilsoftware , our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

Can you please share your application code here? It is really hard to infer details about you setup without usage display.

**Reply by vladi-stevanovic** [reply]
👋 Hello @senthilsoftware I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
